controller/registries/consul/config_test.go

```diff
@@ -50,28 +55,45 @@ func TestStart(t *testing.T) {
 		logger: log.NewLogger(&log.RegistryLoggerOptions{
 			Name: "test",
 		}),
-		softDeletedServices: map[consulService]bool{},
-		done:                make(chan struct{}),
-		watchingServices:    map[consulService]bool{},
-	}
-	config := &consul.Config{
-		ServerUrl: "http://127.0.0.1:8500",
+		done: make(chan struct{}),
 	}
 
-	err := reg.Start(config)
-	assert.NoError(t, err)
+	Convey("Test Start method", t, func() {
+
+		patches := gomonkey.ApplyMethod(reflect.TypeOf(reg), "FetchAllServices", func(_ *Consul, client *Client) (map[consulService]bool, error) {
```

Comment:
We don't need to make this method public. See https://github.com/mosn/htnn/blob/78c9bf58432b0485f4a3f11094e6b9c411fd4ff4/controller/registries/nacos/config_test.go#L94